cases/src/misc.rs

```diff
@@ -269,3 +269,71 @@ pub fn compress_into_bits(data: &[u8]) -> [u8; 256] {
     }
     res
 }
+
+pub fn greater_i256(l: &[u8], r: &[u8]) -> bool {
+    if l == r {
+        return false;
+    }
+
+    let l_s = get_bit_in_slice(l, 255);
+    let r_s = get_bit_in_slice(r, 255);
+
+    if l_s != r_s {
+        if l_s == 0 {
+            true
+        } else {
+            false
+        }
+    } else {
+        let l = U256::from_little_endian(l);
+        let r = U256::from_little_endian(r);
+        l > r
+    }
+}
```

Comment:
This implementation is wrong: what if they are both negative numbers?

Reply:
When 16bit, 0xFFFF is -1, 0xFFFE is -2

Reply:
It's OK. But I think we can use eint instead now.